packages/mobile/src/pincode/PincodeSet.tsx

```diff
@@ -118,22 +124,24 @@ export class PincodeSet extends React.Component<Props, State> {
   render() {
     const { route } = this.props
     const isVerifying = route.params?.isVerifying
+    const changePin = route.params?.changePin
+
     const { pin1, pin2, errorText } = this.state
 
     return (
-      <SafeAreaView style={styles.container}>
+      <SafeAreaView style={changePin ? styles.changePinContainer : styles.container}>
         <DevSkipButton onSkip={this.navigateToNextScreen} />
         {isVerifying ? (
-          // Verify
           <Pincode
+            title="Enter PIN again to confirm"
```

Comment:
These texts should be translated. Also, this is changing the onboarding flow, do we want them to show up always or only if `changePin` is true?

Reply:
i think that the text is shared, according to this figma. is this the only change you're talking about, the title? 
![image](https://user-images.githubusercontent.com/12700801/114483557-717be000-9bbd-11eb-9a34-442e17eefe33.png)

Reply:
https://www.figma.com/file/JHrjTQVHlAPnNTzcoEGeVy/Wallet-4.0?node-id=11117%3A2318 figma link here

Reply:
Ok, sounds good, the `verifying` text seems to be shared but the initial one isn't (`Create a new PIN` vs `Change your PIN` or something like that).

Also, needs to be on a locales file to be translated

Reply:
oh i see, ill put a conditional looks like originally it's just empty, so will just check that changePin prop is true to display this title.

How do the locales files work? Which file should i add this title to?

Reply:
lmk if that worked, i changed the string to be the translated version